src/main/resources/test_files/runtime-spark/JsonFile.jq

```diff
@@ -1,2 +1,2 @@
 (:JIQS: ShouldRun; Output="({ "guess" : "Latvian", "target" : "Russian", "country" : "AU", "choices" : [ "Lao", "Latvian", "Russian", "Swahili" ], "sample" : "b7df3f9d67cef259fbcaa5abcad9d774", "date" : "2013-08-20" }, { "guess" : "Russian", "target" : "Russian", "country" : "AU", "choices" : [ "Croatian", "Nepali", "Russian", "Slovenian" ], "sample" : "8a59d48e99e8a1df7e366c4648095e27", "date" : "2013-08-20" }, { "guess" : "Czech", "target" : "Czech", "country" : "SE", "choices" : [ "Maori", "Czech", "Korean", "Turkish" ], "sample" : "1787b5c79a00b3513ce76847bc1f5b75", "date" : "2013-08-20" }, { "guess" : "Greek", "target" : "Serbian", "country" : "SE", "choices" : [ "German", "Greek", "Kannada", "Serbian" ], "sample" : "0d5b697ebb326b5043ce7fa60a7b968d", "date" : "2013-08-20" }, { "guess" : "Serbian", "target" : "Serbian", "country" : "AU", "choices" : [ "Dari", "Serbian", "Sinhalese", "Vietnamese" ], "sample" : "0d5b697ebb326b5043ce7fa60a7b968d", "date" : "2013-08-20" })" :)
-json-file("./src/main/resources/queries/conf-ex.json", 10)
\ No newline at end of file
+json-file("./src/main/resources/queries/conf-ex.json", 10)
```

Comment:
Note for later (the review reminded me of this): could we add, at some point (nothing urgent), tests on empty files, to check that nothing crashes and an empty sequence is, indeed, returned?

Reply:
I have added the tests for reading the empty files. The parquet file does not have the emptyFile test however (it is disabled) as it doesn't fit into the parquet format. What kind of a test would you like to add there ?